central/notifiers/email/email.go

```diff
@@ -256,6 +256,8 @@ func (m Message) writeContentBytes(buf *bytes.Buffer) {
 
 	writer := multipart.NewWriter(buf)
 	boundary := writer.Boundary()
+	var idx int
+	numberOfAttachments := len(m.Attachments)
```

Comment:
I think we can have `multipart/mixed` content type even if there are 0 attachments. So, instead of `numAttachments`, you can use a `bool mixedType` like below
```
var bool mixedType
if m.EmbedLogo || len(m.Attachments) > 0 {
    mixedType = true
}
if mixedType {
    buf.WriteString(fmt.Sprintf("Content-Type: multipart/mixed; boundary=\"%s\"\r\n", boundary))
    buf.WriteString(fmt.Sprintf("\n--%s\r\n", boundary))
}
```

Reply:
@charmik-redhat Thanks for reviewing. I like your suggestion and will update the code accordingly. Additionally, I will add test cases with multiple attachments and no attachments, but with logo. Consequently we will have more test coverage.